src/Sulu/Bundle/PageBundle/Admin/PageAdmin.php

```diff
@@ -337,8 +337,9 @@ public function configureViews(ViewCollection $viewCollection): void
             );
             $viewCollection->add(
                 $this->viewBuilderFactory
-                    ->createFormViewBuilder('sulu_page.page_edit_form.permissions', '/permissions')
+                    ->createPreviewFormViewBuilder('sulu_page.page_edit_form.permissions', '/permissions')
                     ->setResourceKey('permissions')
+                    ->setPreviewResourceKey(BasePageDocument::RESOURCE_KEY)
```

Comment:
Would do the same für `Activity & Versions` then, so that every tab has a preview

Reply:
@luca-rath will do when the change was accept

Reply:
@luca-rath started with a PR for that - not that easy because we hav no "PreviewList" or "PreviewResourceTabs" view https://github.com/sulu/sulu/pull/6270